apps/system/lockscreen/js/lockscreen.js

```diff
@@ -1091,6 +1097,28 @@
     this.lockScreenClockWidget.start();
   };
 
+  LockScreen.prototype.fetchLockedInterval = function() {
```

Comment:
Since it doesn't do simple 'get' which usually means only getting value without do any effects, I would keep the prefix. Or we need  to call it 'getAndMaybeUpdate...' which is too verbose.